backend/prompt_studio/prompt_studio_core/exceptions.py

```diff
@@ -86,3 +88,10 @@ class PromptNotRun(APIException):
         "it can be used as a variable in another prompt. "
         "Please execute the prompt first and try again."
     )
+
+    def __init__(self, detail: Optional[str] = None, code: Optional[int] = None):
+        if detail is not None:
+            self.detail = detail
+        if code is not None:
+            self.code = code
+        super().__init__(detail, code)
```

Comment:
@harini-venkataraman this is not required - it will be inherited from APIException anyway, can you take care of removing this later?